UI Session/UI Session 9/js/index.js

```diff
@@ -0,0 +1,259 @@
+            break;
+        
+        case 4:
+            isFieldValid = validateConfirmPassword(prefix + fieldNumber, prefix + (fieldNumber - 1));
+            break;
+        
+        case 5:
+            isFieldValid = validateField(prefix + fieldNumber, contactNumberPattern);
+            break;
+
+        default:
+            console.log("Id mismatch"); 
+    }
+    if (fieldNumber >= 6) {
+        isFieldValid = true;
+    }
+    return isFieldValid;
+}
+
+function showNextInputField() {
+    if (count <= totalField) {
+        // To get the name for message.
+        if (count == 2) {
+            name = document.getElementById(prefix + (count-1)).value;
+            message = 'Hi ' + name + ', Can I know your ';
+        }
+        console.log('Count : ' + count);
+        // To get the vehicle type.
+        if (count == 9) {
+            let vehicleTypeField = document.getElementById(prefix + (count - 1));
+            vehicleType = vehicleTypeField.options[vehicleTypeField.selectedIndex].value;
+            console.log(vehicleType);
+        }
+        document.getElementById(prefix + (count-1)).parentElement.style.display = 'none';
+        let nextField = document.getElementById(prefix + count).parentElement;
+        nextField.style.display = 'block';
+        // Getting the name of next field
+        let nextFieldNameArray = nextField.id.split('-');
+        let nextFieldFullName = '';
+
+        // To get the full field name for e.g. 'confirm' + 'password' will become 'confirm password'.
+        for (let index = 1; index < nextFieldNameArray.length; index++) {
+            nextFieldFullName += nextFieldNameArray[index] + ' ';
+        }
+        // Setting the message in the paragraph tag before field.
+        nextField.children[0].innerHTML = message + nextFieldFullName;
+        count++;
+    }
+}
+
+function enterKeyHandler(event) {
+    console.log("key pressed");
+    let target = event.currentTarget;
+    let tag = target.tagName;
+    let char = event.char || event.charCode || event.which;
+    let s = String.fromCharCode(char);
+    if (findAndValidateField((count-1))) {
+        if (char == 13) {
+            console.log('Enter Clicked...');
+            showNextInputField();
+        }
+    }
+}
+
+function switchForm(currentFormId, nextFormId) {
+    console.log(isEmployeeDetailsValid);
+    if (isEmployeeDetailsValid) {
+        document.getElementById(nextFormId).style.display = 'block';
+        document.getElementById(currentFormId).style.display = 'none';
+        document.getElementById('empId').innerHTML = 'You have successfully registered with employee id : ' + empId;
+        empId++;
+        count++;
+    }
+}
+
+function showPassInfo() {
+    document.getElementById('vehicle-form').style.display = 'none';
+    document.getElementById(vehicleType).style.display = 'block';
+    currentPassPrice = getPassPrice();
+}
+
+function getPassPrice() {
```

Comment:
Can we create a function there we can pass data according to condition and that function can assign values.